Modules/_sqlite/connection.c

```diff
@@ -1478,6 +1478,94 @@ pysqlite_connection_iterdump(pysqlite_Connection* self, PyObject* args)
 }
 
 static PyObject *
+pysqlite_connection_backup(pysqlite_Connection* self, PyObject* args, PyObject* kwds)
```

Comment:
If I read [SQLite release notes](https://www.sqlite.org/changes.html) correctly, `sqlite3_backup_init` has been added in 3.6.11 so we need to expose this only if SQLite version is 3.6.11 or newer.